candig/server/cli/repomanager.py

```diff
@@ -180,6 +181,27 @@ def addDataset(self):
         dataset.setAttributes(json.loads(self._args.attributes))
         self._updateRepo(self._repo.insertDataset, dataset)
 
+    def addDatasetDuo(self):
+        """
+        Adds DUO info to existing dataset into this repo.
+        """
+        self._validateRepo()
+        dataset = datasets.Dataset(self._args.datasetName)
+
+        with open(self._args.dataUseOntologyFile, 'r') as f:
```

Comment:
Might want to throw some errors like IOError or if json load fails.

Reply:
Fixed now!

![image](https://user-images.githubusercontent.com/18109953/70193699-cf88e100-16b4-11ea-8268-47184d472de9.png)